kevm_pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -101,18 +101,27 @@ def _get_method_abi(_mname: str) -> Dict:
 
         self.name = contract_name
         self.bytecode = (contract_json['evm']['deployedBytecode']['object'] if not foundry else contract_json['deployedBytecode']['object'])
-        method_identifiers = contract_json['evm']['methodIdentifiers'] if not foundry else contract_json['methodIdentifiers']
+        try:
+            method_identifiers = contract_json['evm']['methodIdentifiers'] if not foundry else contract_json['methodIdentifiers']
+        except KeyError:
+            _LOGGER.error(f'\'methodIdentifiers\' not found on contract {self.name}')
+            method_identifiers = []
```

Comment:
Same here, it seems we can do this with an if-else and a `_LOGGER.warning` instead.

Reply:
Done.